Description: Hard coded stripping behaviour.
 Remove explicit stripping from makefile target.
X-Closes: #437615
X-Comment: debdiff netkit-ftp_0.17-{12,16}.dsc
Author: Alberto Gonzalez Iniesta <agi@inittab.org>
Forwarded: no
Last-Update: 2008-03-21
diff -u netkit-ftp-0.17/ftp/Makefile netkit-ftp-0.17/ftp/Makefile
--- netkit-ftp-0.17/ftp/Makefile
+++ netkit-ftp-0.17/ftp/Makefile
@@ -16,7 +16,7 @@
 cmds.o glob.o: glob.h
 
 install: ftp
-	install -s -m$(BINMODE) ftp $(INSTALLROOT)$(BINDIR)
+	install $(INSTALL_OPTS) -m$(BINMODE) ftp $(INSTALLROOT)$(BINDIR)
 	ln -sf ftp $(INSTALLROOT)$(BINDIR)/pftp
 	install -m$(MANMODE) ftp.1 $(INSTALLROOT)$(MANDIR)/man1
 	ln -sf ftp.1 $(INSTALLROOT)$(MANDIR)/man1/pftp.1
